src/deploy/osp_deployer/director.py

```diff
@@ -706,17 +709,44 @@ def configure_dhcp_server(self):
     def setup_networking(self):
         logger.debug("Configuring network settings for overcloud")
 
+        if self.settings.enable_ovs_dpdk:
+            mode = str(self.settings.ovs_dpdk_mode)
+            compute_file_name = "compute-ovs-dpdk-mode" + mode + ".yaml"
         static_ips_yaml = self.templates_dir + "/static-ip-environment.yaml"
         static_vip_yaml = self.templates_dir + "/static-vip-environment.yaml"
+        neutron_ovs_dpdk_yaml = self.templates_dir + "/neutron-ovs-dpdk.yaml"
 
         # Re - Upload the yaml files in case we're trying to
         # leave the undercloud intact but want to redeploy
         # with a different config
         self.upload_file(self.settings.static_ips_yaml, static_ips_yaml)
         self.upload_file(self.settings.static_vip_yaml, static_vip_yaml)
+        self.upload_file(self.settings.neutron_ovs_dpdk_yaml,
+                         neutron_ovs_dpdk_yaml)
 
         self.setup_nic_configuration()
 
+        if self.settings.enable_ovs_dpdk is True:
```

Comment:
If we agree that we should not restrict the DPDK feature to a specific NIC configuration on only 2 server models, then this code (729-749) should also be removed.